app/_src/gateway/plugin-development/pdk/index.md

```diff
@@ -12,9 +12,16 @@ source_url: https://github.com/Kong/kong/tree/master/kong/pdk
 ---
 
 The Plugin Development Kit (PDK) is set of Lua functions and variables
- that can be used by plugins to implement their own logic.  The PDK is a
+ that can be used by plugins to implement their own logic.
+ {% if_version lte:3.1.x %}
```

Comment:
```suggestion
 {% if_version lte:3.0.x %}
```
I believe this was actually changed in 3.1, we just didn't update the PDK docs then.